### PR TITLE
[Merged by Bors] - feat: add smdk publish

### PR DESCRIPTION
**Pull request by digikata**
part of #2631

### CODE REVIEW

File: crates/fluvio-hub-util/src/keymgmt.rs [hunk 1]
```diff
@@ -109,24 +102,33 @@ impl Keypair {
...
+        Keypair::from_secret_bytes(&pkbytes)
+    }
+
     /// a pubkey from an ssh private key (id_ed25519) generated via
     ///  e.g. $ ssh-keygen -t ed25519 -C "sshkey@example.com" -f ./id_ed25519 -P ""
     /// ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIACGeGHWvt/E60k/FLuDsCkArLAIa4lvwk1wg3nJIGJl sshkey@example.com
     pub fn from_ssh(env_val: &str) -> Result<Keypair> {
         let sshprivkey =
-            ssh_key::PrivateKey::from_openssh(&env_val).map_err(|_| HubUtilError::KeyVerify)?;
+            ssh_key::PrivateKey::from_openssh(&env_val).expect("reading ssh pub key file");
```
Comment (sehz) [inline crates/fluvio-hub-util/src/keymgmt.rs:115]: why expect? `map_err` seems better

**Review by sehz — APPROVED** [review]
A minor nit, but we can fix post commit

File: crates/fluvio-hub-util/src/keymgmt.rs (hunk 1 above)
Comment (digikata) [reply crates/fluvio-hub-util/src/keymgmt.rs:115]: I'll add the change to the download pr

**Pull request closed without merging**